<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestForm</class>
 <widget class="QDialog" name="TestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Designer Sample</string>
  </property>
  <widget class="QLabel" name="eng_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>영어</string>
   </property>
  </widget>
  <widget class="QLabel" name="kor_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>국어</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="math_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>수학</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_let">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>17</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="kor_let">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>49</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="eng_let">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>80</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="math_let">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>107</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="total_pbt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>합계</string>
   </property>
  </widget>
  <widget class="QPushButton" name="avg_pbt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>평균</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_let">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="avg_let">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
